### PR TITLE
[12.0][ADD] add new addon attribute_set_mass_edit

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**

### CODE REVIEW

File: attribute_set_mass_edit/models/attribute_attribute.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    @api.multi
+    def unlink(self):
+        for attribute in self:
+            mass_object = attribute._get_mass_object()
+            if mass_object:
+                attribute._remove_attribute_from_mass_object(mass_object)
+        return super(AttributeAttribute, self).unlink()
+
+    @api.model
+    def create(self, vals):
```
Comment (lmignon) [inline attribute_set_mass_edit/models/attribute_attribute.py]: @Cedric-Pigeon With this implementation you breaks the possibility for batch creation... https://github.com/odoo/odoo/blob/12.0/odoo/models.py#L3490

We should pay attention to try to always support batch processing in all the methods.

**Review by sebastienbeau — APPROVED** [review]
Great feature, code review

**Pull request closed without merging**